x/oracle/keeper/msg_server.go

```diff
@@ -128,6 +128,52 @@ func (ms msgServer) AggregateExchangeRateVote(goCtx context.Context, msg *types.
 	return &types.MsgAggregateExchangeRateVoteResponse{}, nil
 }
 
+func (ms msgServer) AggregateExchangeRateCombinedVote(goCtx context.Context, msg *types.MsgAggregateExchangeRateCombinedVote) (*types.MsgAggregateExchangeRateCombinedVoteResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	valAddr, err := sdk.ValAddressFromBech32(msg.Validator)
+	if err != nil {
+		return nil, err
+	}
+
+	params := ms.GetParams(ctx)
+
+	var voteErr error
+	aggregatePrevote, err := ms.GetAggregateExchangeRatePrevote(ctx, valAddr)
+	// if there isn't a prevote, we want to no-op the vote so we don't get an error
+	// this way, it is safe to use combined vote regardless of a missed vote window
+	if err == nil && (uint64(ctx.BlockHeight())/params.VotePeriod)-(aggregatePrevote.SubmitBlock/params.VotePeriod) == 1 {
```

Comment:
nit we can put `(uint64(ctx.BlockHeight())/params.VotePeriod)-(aggregatePrevote.SubmitBlock/params.VotePeriod) == 1` to a helper function called something like `IsPrevoteCurrent` for readability

Reply:
done